### PR TITLE
Change device iframe URL to absolute path(starts with '/')

### PR DESCRIPTION
**Pull request by k311093**
Using window.location.pathname as prefix to make possible to serve normally under proxy.

### CODE REVIEW

**Comment by ikicha** [comment]
@jemoreira , can you test if it meets your requirements?

File: frontend/src/operator/webui/src/app/safe-device-url.pipe.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class SafeDeviceUrlPipe implements PipeTransform {
   transform(deviceId: string) {
     /* eslint-disable */
     // DO NOT FORMAT THIS LINE
-    return this.sanitizer.bypassSecurityTrustResourceUrl(`devices/${deviceId}/files/client.html`);
+    return this.sanitizer.bypassSecurityTrustResourceUrl(`/${window.location.pathname}devices/${deviceId}/files/client.html`);
```
Comment (jemoreira) [inline frontend/src/operator/webui/src/app/safe-device-url.pipe.ts:13]: I expected this to be `${window.location.pathname}/devices...`.
For every page I tested, the `pathname` value starts with a `/`, but not all end with one so I expected to see `/` between `${window.location.pathname}` and `devices`. Some pages do end in `/`, but the browser normally ignores `//` and treats it like a single `/`.

Comment (k311093) [reply frontend/src/operator/webui/src/app/safe-device-url.pipe.ts]: I tried to use this under proxy suburl. It seems like have a problem with window.location.pathname I'll check it further and will update the PR again.

**Review by ikicha — APPROVED** [review]

**Review by ikicha — CHANGES_REQUESTED** [review]
As @k311093 said in offline, we need to find a way to support both internal and external.

**Pull request closed without merging**